### DISCUSSION TITLE
[Question] useDialog 사용시 dialog의 위치

### DISCUSSION THREAD

**Discussion by yangSangHoon** [wanteddev/montage-web#35] [Q&A]
### 요약

dialog가 현재는 중간에 뜨는 것 같은데 -> 임의로 left, top등을 지정 할수 있을지 문의 드립니다.

### 상세

예시)
case1. dialog가 모바일일때 상단 20px에서 노출 희망

case2. 컨턴츠 넓이가 브라우져사이즈보다 클 경우
![Screenshot 2024-06-04 at 9 16 44 AM](https://github.com/wanteddev/wds/assets/15603335/98f5d2f1-6a5c-4dad-9c60-b5b54f4b28a0)

**Comment by Sh031224** [comment]
이건 버그로 보는게 맞는 것 같은데, 모바일일 때 아래와 같이 보이면 괜찮을까요?

<img width="379" alt="image" src="https://github.com/wanteddev/wds/assets/51149996/105eeecd-f59d-443f-b70c-a932079bbd3e">

**Reply by yangSangHoon** [reply]
오 넵 감사합니다!~
